db/seeds/dev/songs.js

```diff
@@ -0,0 +1,17 @@
+exports.seed = async function (knex) {
+  try {
+    await knex('genres').del() // delete all genres first
+    await knex('songs').del() // delete all songs
+
+    // Now that we have a clean slate, we can re-insert our songs data
+    // Insert a single song, return the song ID, insert 2 genres
```

Comment:
Will need to remove these comments at some point but I would agree with leaving them there for clarity while we are finishing up the backend seeding